.github/workflows/3x.yml

```diff
@@ -88,18 +87,15 @@ jobs:
           coverage: pcov
           ini-values: error_reporting=E_ALL
       - name: Download repo content from artifact
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: full-workspace
       - name: Download terminus.phar as artifact
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
           name: terminus-phar
       - name: Install Composer Dependencies
         run: composer install --no-interaction --prefer-dist
-      - name: Setup tmate session
```

Comment:
Please do not remove tmate

Reply:
Agree

Reply:
k. I remember adding it a year or so ago and then not using it. Adding it back.